v3/cypress/e2e/plugin.spec.ts

```diff
@@ -1,3 +1,4 @@
+/// <reference types="cypress" />
```

Comment:
I found myself annoyed by this as well and submitted a PR to fix it, so you shouldn't need this line any more.